thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js

```diff
@@ -282,6 +282,29 @@ export function buildAnomalyStats(alertEvalMetrics, anomalyStats, showProjected
   return anomalyStats;
 }
 
+/**
+ * Caches duration data to local storage in order to persist it across pages
+ * TODO: Move this to self-serve services
+ * @method setDuration
+ * @param {String} duration - qualifies duration set as 'custom' or a range key like '3m'
+ * @param {Number} startDate - start date for alert page (and list of anomalies)
+ * @param {Number} endDate - end date for alert page (and list of anomalies)
+ * @return {undefined}
+ */
+export function setDuration(duration, startDate, endDate) {
+  localStorage.setItem('duration', JSON.stringify({ duration, startDate, endDate }));
```

Comment:
This should support all newer version of the browsers we use. Looks like there's no  need for a polyfill/addon at this point. 

<img width="944" alt="screen shot 2018-02-14 at 10 24 49 am" src="https://user-images.githubusercontent.com/8664954/36220999-7ea2997e-1171-11e8-9202-11d8ce796c96.png">

Reply:
Thanks for adding that note.